tests/analyzer_integration_broken_tests.txt

```diff
@@ -196,3 +196,4 @@ test_quota/test.py::test_tracking_quota
 test_quota/test.py::test_users_xml_is_readonly
 test_replicating_constants/test.py::test_different_versions
 test_merge_tree_s3/test.py::test_heavy_insert_select_check_memory[node]
+test_wrong_db_or_table_name/test.py::test_wrong_table_name
```

Comment:
why do we need this?

Reply:
Me and Nikolai considered this test is not able to be checked by analyzer yet, so it’s not checking by analyzer now.